extension-manifest-v3/scripts/build.js

```diff
@@ -174,6 +212,10 @@ if (manifest.permissions.includes('declarativeNetRequest') && argv.watch) {
   manifest.permissions.push('declarativeNetRequestFeedback');
 }
 
+manifest.web_accessible_resources.push({
+  resources: Array.from(seenWebAccessibleResources),
```

Comment:
how about adding a whole `web_accessible_resources` folder to all manifests? that way we wont have to change the manifest in the build time

Reply:
Moved to `web_accessible_resources/statics`